### PR TITLE
Nerve Gas for Marauders & Some Pirates

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Feature**

This PR addresses the bug/feature described in issues #10204 and #2218.

## Summary
I added Nerve Gas (sorry for the capitalization) to three especially pirate ships. I can remove it from the Modified Argosy if people want, because that is a bit iffy. I’m also requesting a review from @Amazinite before this gets merged. 
Important to note is that the Nighthawk also has Nerve Gas, which I did _not_ add. 

## Screenshots
None. 

## Usage examples
N/A

## Testing Done
No. 

## Save File
This save file can be used to test these changes:
None. Start a World Forge or Omnis game with this PR installed and buy a ship. 

## Artwork Checklist
N/A

## Wiki Update
N/A

## Performance Impact
N/A

### CODE REVIEW

**Comment by Quantumshark** [comment]
I might add it to one or two of the Marauders as well, potentially? Other than that, looks good.

**Comment by TheGiraffe3** [comment]
Forgot about the Marauders. I’ll get to them in a second.

**Comment by TheGiraffe3** [comment]
@Quantumshark do you think it’s a good idea for the Modified Argosy to have Nerve Gas, or is that just to common to be used?

**Comment by Saugia** [comment]
Looks good so far; I agree with Quantum. Was looking at the issue first, but yeah, Marauders sound like a good target here.

File: data/human/ships.txt [hunk 1]
```diff
@@ -2752,7 +2753,8 @@ ship "Modified Argosy"
 		"D23-QP Shield Generator"
 		"Small Radar Jammer"
 		"Brig"
-		"Laser Rifle" 2
+		"Laser Rifle" 12
+		"Nerve Gas"
```
Comment (TheGiraffe3) [inline data/human/ships.txt:2757]:
```suggestion
```

**Comment by TheGiraffe3** [comment]
I think I’ll just remove Nerve Gas from the Modified Argosy, because that shows up quite frequently in the South.

**Comment by Quantumshark** [comment]
You could maybe vary the numbers on the Marauders a little more, and not give nerve gas to all three variants in each case? 
Particularly with the heavy warships, it'd create a slightly more interesting challenge when capturing one if one or two variants came stocked with a full complement of nerve gas.
As far as I'm aware, that'd be the first case of an NPC having more attack than defense, which would be a positive for making boarding a touch more varied.

**Comment by TheGiraffe3** [comment]
The way I was currently thinking was Weapons has more H2H, Engines has less. But I’ll update it so that some do have a full complement.

**Comment by TheGiraffe3** [comment]
@Quantumshark thoughts on the one-crew Marauders having Nerve Gas?

**Comment by Quantumshark** [comment]
Maybe just the crazier ones. It seems appropriate for the Engines Arrow, I think.

**Comment by TheGiraffe3** [comment]
Added it to the Engines Arrow, and also the Engines Quicksilver.

File: data/human/marauders.txt [hunk 2]
```diff
@@ -300,6 +302,7 @@ ship "Marauder Falcon" "Marauder Falcon (Engines)"
 		"Large Radar Jammer"
 		"Outfits Expansion" 4
 		"Laser Rifle" 80
+
```
[the remark this answers is not in the document]
Comment (TheGiraffe3) [reply data/human/marauders.txt]: Whoops.

**Review by Quantumshark — APPROVED** [review]

**Comment by nickshanks** [comment]
What does "World Forge or Omnis game" mean?

**Comment by TheGiraffe3** [comment]
There are two plugins, [World Forge](https://github.com/EndlessSkyCommunity/world-forge) and [Omnis](https://github.com/EndlessSkyCommunity/-Omnis), which are extensively used for testing.

**Comment by TheGiraffe3** [comment]
@tibetiroka do you want to remove the `to be merged` label?

**Pull request merged**